sycl/test-e2e/KernelFusion/Reduction/group_reduce_and_last_wg_detection.cpp

```diff
@@ -0,0 +1,9 @@
+// RUN: %{build} -fsycl-embed-ir -o %t.out
+// RUN: %{run} %t.out
+// UNSUPPORTED: hip || cuda
```

Comment:
Add a comment (`COM:`) why this is unsupported.

Reply:
Done